Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -19,7 +31,16 @@ public static void Sort(int[] array)
         /// <param name="array">Input array in a random order.</param>
         public static void Reverse(int[] array)
         {
-            // ToDo: implement.
+            if (array == null || array.Length == 0) return;
+
+            var nextIndex = array.Length - 1;
+            for (var i = 0; i < nextIndex; i++)
+            {
+                var tempNumber = array[i];
```

Comment:
Can be extracted to a ``Swap`` method because it's repeated in both shuffle and sort.